openwisp_radius/tests/test_api/test_phone_verification.py

```diff
@@ -81,7 +81,7 @@ def test_register_201(self):
         user = User.objects.get(email=self._test_email)
         self.assertTrue(user.is_member(self.default_org))
         self.assertEqual(user.phone_number, phone_number)
-        self.assertFalse(user.is_active)
+        self.assertTrue(user.is_active)
```

Comment:
Good that we spotted it!
So if here we were expecting the is_active to be False, it means the user was not verified, so can you please add another assertion to ensure that?
Should apply to the other occurrences as well.